Update html.rst (#6131)
cakephp/docs: en/views/helpers/html.rst

    echo $this->Html->image('Blog.icon.png', ['plugin' => false]);

> Last but no least, if you would like the prefix of the url NOT to be /img, you could override this setting by specifying the prefix on the ``$options`` array ::

Reviewer: ```suggestion
If you would like the prefix of the URL to not be ``/img``, you can override this setting by specifying the prefix in the ``$options`` array ::
```
